.github/workflows/scan-rock.yaml

```diff
@@ -0,0 +1,62 @@
+name: Scan
+
+on:
+  workflow_call:
+    inputs:
+      rock-filename:
+        description: "Name of the .rock file inside the artifact to be scanned"
+        required: true
+        type: string
+jobs:
+  scan:
+    name: Scan ${{ inputs.rock-filename }}
+    runs-on: ubuntu-22.04
+    steps:
+      # Ideally we'd use self-hosted runners, but this effort is still not stable.
+      # This action will remove unused software (dotnet, haskell, android libs, codeql,
+      # and docker images) from the GH runner, which will liberate around 60 GB of storage
+      # distributed in 40GB for root and around 20 for a mnt point.
+      # We need it to avoid cases where scanning fails due to "no space left on device". 
+      - name: Maximise GH runner space
+        uses: easimon/maximize-build-space@v7
+        with:
+          root-reserve-mb: 40960
+          remove-dotnet: 'true'
+          remove-haskell: 'true'
+          remove-android: 'true'
+          remove-codeql: 'true'
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: artifact (${{ inputs.rock-filename }})
+
+      - name: Export ROCK to Docker
+        id: rock_in_docker
+        run: |
+          sudo skopeo --insecure-policy copy oci-archive:${{ inputs.rock-filename }} docker-daemon:rock:tag
+          echo "image=rock:tag" >> "$GITHUB_OUTPUT"
+
+      - name: Log in to Docker Hub
```

Comment:
can you confirm if a log in step is required here?

Reply:
Confirmed that we do NOT need it, thus removing it.